js/client-teambuilder.js

```diff
@@ -398,7 +398,7 @@
 					// support dragging and dropping buttons.
 					buf += '<li><div name="edit" data-value="' + i + '" class="team" draggable="true">' + formatText + '<strong>' + Tools.escapeHTML(team.name) + '</strong><br /><small>';
 					buf += Storage.getTeamIcons(team);
-					buf += '</small></div><button name="edit" value="' + i + '"><i class="fa fa-pencil"></i>Edit</button><button name="delete" value="' + i + '"><i class="fa fa-trash"></i>Delete</button></li>';
+					buf += '</small></div><button name="edit" value="' + i + '"><i class="fa fa-pencil"></i></button><button name="newTop" value="' + i + '"><i class="fa fa-clone"></i></button><button name="delete" value="' + i + '"><i class="fa fa-trash"></i></button></li>';
```

Comment:
I'm not sure if this is the right place to put a button...

Maybe a "More" menu.

Reply:
IMHO it looks pretty good like this

![screen shot 2017-12-02 at 19 35 41](https://user-images.githubusercontent.com/8976828/33521151-13cacefe-d798-11e7-8976-e7ecb006c02f.png)

Reply:
Icons without labels are pretty useless. I wouldn't recommend them.

Reply:
hmm, then perhaps shorten the width of the teams?

![screen shot 2017-12-02 at 22 49 50](https://user-images.githubusercontent.com/8976828/33522139-20e20416-d7b3-11e7-99c5-7eec5ff6ecf9.png)

Reply:
I don't know if "Clone" needs top billing among "Edit" and "Delete"

Hence a "More" menu. Maybe just a dropdown arrow to the right of Delete, or to the left of Edit. It would take some space but not nearly as much.

Reply:
Or just have a More menu that replaces "Edit". Clicking on the team does edit it.

Reply:
I think having a more menu with only one option is kinda pointless, you might as well have just the option at the top level...

if someone thinks up of other things to put in such a menu that would work

Reply:
A menu item to validate the team?

Reply:
Edit, Clone is at least two.

Reply:
I don't see the point in an edit button...

Reply:
Mostly, in case people aren't thinking and forget that clicking on the big button means "Edit".